pkg/services/oracle/request.go

```diff
@@ -126,6 +127,11 @@ func (o *Oracle) processRequest(priv *keys.PrivateKey, req request) error {
 			}
 			switch r.StatusCode {
 			case http.StatusOK:
+				if !checkMediaType(r.Header.Get("Content-Type"), o.MainCfg.AllowedContentTypes) {
+					resp.Code = transaction.ContentTypeNotSupported
```

Comment:
Don't we want to log a warning? It's done for ProtocolNotSupported.

Reply:
`ProtocolNotSupported` is an invalid request, `ContentTypeNotSupported` is an invalid reply. We usually don't log these (it's not much different from Forbidden/NotFound/...).